ecorest/config/main.go

```diff
@@ -2,19 +2,46 @@ package config
 
 import (
 	"github.com/OpenTreeMap/otm-ecoservice/eco"
+	"os"
 )
 
+type server struct {
```

Comment:
Oh woops. Gone in https://github.com/maurizi/otm-ecoservice/commit/1d614340e63070434c403f751fcaceeb0f91e30d